src/Microsoft.AspNetCore.Blazor.Mono/HowToUpgradeMono.md

```diff
@@ -43,11 +44,21 @@ At this stage, make a Git commit with a message similar to `Upgrade Mono binarie
 
 Push your change as a PR to the `blazor` repo. Once the CI system has built the resulting package, download it. It usually has a name like `Microsoft.AspNetCore.Blazor.Mono.3.2.0-ci.nupkg`.
 
-Switch over to the `aspnetcore` repo and modify the `eng\Versions.props` file to use your new version (e.g., `3.2.0-ci`). To restore this correctly,
-
+Switch over to the `aspnetcore` repo and modify the `eng\Versions.props` file to use your new version (e.g., `3.2.0-ci`). Locate the `MicrosoftAspNetCoreBlazorMonoPackageVersion` property and update the version to the desired version:
+ ```<MicrosoftAspNetCoreBlazorMonoPackageVersion>3.2.0-CI</MicrosoftAspNetCoreBlazorMonoPackageVersion>```
+ 
+ To restore this correctly:
+ 
  * Make sure you don't already have a `-ci` version of that package in your NuGet package cache
  * Run a `dotnet restore` with `-s <directory_containing_the_download>`
 
-Now try running the StandaloneApp and perf benchmarks to be sure the apps still work, and the size and perf numbers haven't regressed badly. Ideally, also run all the E2E tests.
+Now try running the StandaloneApp and perf benchmarks to be sure the apps still work, and the size and perf numbers haven't regressed badly. 
+
+After running the above checks, also run all the E2E tests:
+- switch to the `src\Components\test\E2ETest`
+- run `yarn install` to install the necessary packages (selenium, chrome debug adapter, ...)
+- if you don't have installed Java yet, do so
+- run `dotnet test`
```

Comment:
Alternatively you can run the E2E tests from VS, but I don't mind whether or not that's mentioned in this doc.

Reply:
I'll include this too, to not be biased